release.md

```diff
@@ -0,0 +1,298 @@
+  }
+}
+
+// Docker
+compilers: {
+  solc: {
+    version: "0.4.22",   // Any published image name
+    docker: true
+  }
+}
+
+// Relative or absolute path to an npm installed solc-js
+compilers: {
+  solc: {
+    version: "/Users/axic/.nvm/versions/node/v8.9.4/lib/node_modules/solc"
+  }
+}
+
+// Optimization and EVM version settings
+compilers: {
+  solc: {
+    settings: {
+      optimizer: {
+        enabled: true, // Default: false
+        runs: 1000     // Default: 200
+      },
+      evmVersion: "homestead"  // Default: "byzantium"
+    }
+  }
+}
+```
+
+### Speed comparison
+Docker and native binary compilers process large contract sets faster than solcjs. If you're just compiling a few contracts at a time, the speedup isn't significant relative to the overhead of running a command (see below). The first time `truffle` uses a docker version there's some extra overhead as it caches the solc version string and a solcjs companion compiler. All subsequent runs should be at full speed.
+
+Times to `truffle compile` on a MacBook Air 1.8GHz, Node 8.11.1
+
+| Project              | # files | solcjs | docker | bin |
+|----------------------|---------:| ------:|--------:|-----------:|
+| truffle/metacoin-box |       3 |   4.4s |   4.4s |      4.7s |
+| gnosis/pm-contracts  |      34 |  21.7s |  10.9s |     10.2s |
+| zeppelin-solidity    |     107 |  36.7s |  11.7s |     11.1s |
+
+
+For help installing a natively built compiler, see the Solidity docs [here](https://solidity.readthedocs.io/en/v0.4.23/installing-solidity.html#binary-packages).
+
+### Contract Profiling
+Truffle has always tried figure out which contracts changed recently and compile the smallest set necessary (with varying degrees of success). It now does this reliably so you may benefit from setting up an `npm` convenience script that looks like this:
+```
+"test": "truffle compile && truffle test"
+```
+
+Many thanks to the Solidity team for making all of the above possible and for their helpful advice over the last year.  🙌 🙌
+
+
+# 2. 🐇 🐇  Web3 1.0  🐇 🐇
+
+`truffle-contract` now uses Web3 1.0 under the hood.  It's nice! The error handling (especially the parameter checking) is really good. And there's lots of new work happening over there  - ENS support is being added and [EthPrize](http://ethprize.io/) is funding an API to make contract packages published to EthPM easily available as well.
+
+We've tried to minimize the number of changes required to transition an existing test suite from Web3 0.x to 1.0. Unfortunately there are some breaking changes due to differences in the way Web3 1.0 formats outputs (see chart below).
+
+The biggest change is that `truffle-contract` now returns numbers as [BN](https://github.com/indutny/bn.js/) by default instead of BigNumber. This is necessary - BigNumber doesn't have the precision required by the EVM. The number libraries share some methods (notably `.toNumber()`) but their arithmetic operations are quite different.
```

Comment:
I'm torn on whether or not we need the backticks when describing a truffle package. Definitely the name seems to work best as all-lowercase. I think the name can just be regular text. Or maybe bold.